PGM/src/main/java/tc/oc/pgm/PGM.java

```diff
@@ -139,6 +141,8 @@ public void onEnable() {
         this.pollManager = new PollManager(this);
         this.pollableMaps = new PollableMaps();
 
+        Bukkit.getScheduler().scheduleSyncRepeatingTask(this, () -> playerVersionLogger.logPlayerVersions(), 20 * 60 * 15, 20 * 60 * 15);
```

Comment:
you should look into @Repeatable and how PGM does scheduling with dependency injection.
This should be moved to the analytics packet, turned into a PluginFacet and bind it in AnalyticsManifest